[staging] perlPackages: bulk update

stigtsp opened the pull request:
###### Motivation for this change

Update Perl modules from CPAN

- [X] Builds on linux x64
- [ ] Builds on darwin x64
- [ ] Review dependencies / changelogs
- [ ] Test against perl 5.32.1 #110654
- [ ] Consider `doCheck=true;` on: `PerlMagick`, ...
  
Not bumped:
- `Future::AsyncAwait` was separately bumped by #112630
- `HTTP::Daemon` #110908, due to impact on cross compiling NixOS (underlying Module::Build issue) #112173 #66741 #106759
- `Gtk3` due to dependency `perlPackages.GlibObjectIntrospection` being marked as broken
- `ExtUtils::CBuilder` since it is in core, and defined twice in perl-packages.nix TODO (0.280235 fixes compilation on darwin with XCode 12, perl 5.32.1 has [0.280234](https://github.com/Perl/perl5/tree/v5.32.1/dist/ExtUtils-CBuilder))
- ~JavaScript::Minifier::XS https://github.com/bleargh45/JavaScript-Minifier-XS/issues/7~

After:
- [ ] Add [nix-update-cpan](https://github.com/stigtsp/nix-update-cpan) to `maintainers/scripts`

stigtsp committed:
[nix-update-cpan] perlPackages.ArrayCompare: 3.0.7 -> 3.0.8

stigtsp committed:
[nix-update-cpan] perlPackages.Appcpm: 0.997000 -> 0.997002

stigtsp committed:
[nix-update-cpan] perlPackages.AppCLI: 0.50 -> 0.52

stigtsp committed:
[nix-update-cpan] perlPackages.AlienBuild: 2.37 -> 2.38

stigtsp committed:
[nix-update-cpan] perlPackages.Appperlbrew: 0.89 -> 0.91

stigtsp committed:
[nix-update-cpan] perlPackages.CarpAssertMore: 1.24 -> 1.26

stigtsp committed:
[nix-update-cpan] perlPackages.BusinessISBN: 3.005 -> 3.006

stigtsp committed:
[nix-update-cpan] perlPackages.Catmandu: 1.2013 -> 1.2015

stigtsp committed:
[nix-update-cpan] perlPackages.Cairo: 1.108 -> 1.109

stigtsp committed:
[nix-update-cpan] perlPackages.BusinessISBNData: 20191107 -> 20210112.002

stigtsp committed:
[nix-update-cpan] perlPackages.CacheFastMmap: 1.54 -> 1.56

stigtsp committed:
[nix-update-cpan] perlPackages.CatalystViewCSV: 1.7 -> 1.8

stigtsp committed:
[nix-update-cpan] perlPackages.CPANPerlReleases: 5.20201120 -> 5.20210123

stigtsp committed:
[nix-update-cpan] perlPackages.ConfigTiny: 2.24 -> 2.26

stigtsp committed:
[nix-update-cpan] perlPackages.Connector: 1.35 -> 1.36

stigtsp committed:
[nix-update-cpan] perlPackages.ConfigMVP: 2.200011 -> 2.200012

stigtsp committed:
[nix-update-cpan] perlPackages.CompressRawBzip2: 2.096 -> 2.100

stigtsp committed:
[nix-update-cpan] perlPackages.ClassObservable: 1.04 -> 2.000

stigtsp committed:
[nix-update-cpan] perlPackages.CompressRawZlib: 2.096 -> 2.100

stigtsp committed:
[nix-update-cpan] perlPackages.CPANPLUS: 0.9908 -> 0.9910

stigtsp committed:
[nix-update-cpan] perlPackages.CryptCBC: 2.33 -> 3.00

stigtsp committed:
[nix-update-cpan] perlPackages.CSSMinifierXS: 0.09 -> 0.13

stigtsp committed:
[nix-update-cpan] perlPackages.DateTimeFormatW3CDTF: 0.07 -> 0.08

stigtsp committed:
[nix-update-cpan] perlPackages.DateTimeCalendarJulian: 0.102 -> 0.103

stigtsp committed:
[nix-update-cpan] perlPackages.DateTimeFormatStrptime: 1.77 -> 1.78

stigtsp committed:
[nix-update-cpan] perlPackages.CryptJWT: 0.029 -> 0.031

stigtsp committed:
[nix-update-cpan] perlPackages.CryptPasswdMD5: 1.40 -> 1.41

stigtsp committed:
[nix-update-cpan] perlPackages.DevelPatchPerl: 2.04 -> 2.08

stigtsp committed:
[nix-update-cpan] perlPackages.DBDCSV: 0.56 -> 0.57

stigtsp committed:
[nix-update-cpan] perlPackages.DBIxSearchBuilder: 1.68 -> 1.69

stigtsp committed:
[nix-update-cpan] perlPackages.DateTimeTimeZone: 2.44 -> 2.47

stigtsp committed:
[nix-update-cpan] perlPackages.DateTimeLocale: 1.28 -> 1.31

stigtsp committed:
[nix-update-cpan] perlPackages.EmailMIMEContentType: 1.024 -> 1.026

stigtsp committed:
[nix-update-cpan] perlPackages.ExtUtilsInstall: 2.18 -> 2.20

stigtsp committed:
[nix-update-cpan] perlPackages.FileLoadLines: 1.01 -> 1.020.2

stigtsp committed:
[nix-update-cpan] perlPackages.Gnome2VFS: 1.083 -> 1.084

stigtsp committed:
[nix-update-cpan] perlPackages.ExtUtilsMakeMaker: 7.56 -> 7.58

stigtsp committed:
[nix-update-cpan] perlPackages.Gnome2Canvas: 1.004 -> 1.006

stigtsp committed:
[nix-update-cpan] perlPackages.Gnome2: 1.047 -> 1.048

stigtsp committed:
[nix-update-cpan] perlPackages.Gnome2Wnck: 0.16 -> 0.18

stigtsp committed:
[nix-update-cpan] perlPackages.Future: 0.46 -> 0.47

stigtsp committed:
[nix-update-cpan] perlPackages.GnuPGInterface: 1.00 -> 1.01

stigtsp committed:
[nix-update-cpan] perlPackages.Gtk2Unique: 0.05 -> 0.07

stigtsp committed:
[nix-update-cpan] perlPackages.Graph: 0.9712 -> 0.9717

stigtsp committed:
[nix-update-cpan] perlPackages.Gtk2GladeXML: 1.007 -> 1.008

stigtsp committed:
[nix-update-cpan] perlPackages.Gtk2TrayIcon: 0.06 -> 0.07

stigtsp committed:
[nix-update-cpan] perlPackages.HTTPCookies: 6.09 -> 6.10

stigtsp committed:
[nix-update-cpan] perlPackages.IOSocketSSL: 2.068 -> 2.069

stigtsp committed:
[nix-update-cpan] perlPackages.IOInteractive: 1.022 -> 1.023

stigtsp committed:
[nix-update-cpan] perlPackages.IOTty: 1.15 -> 1.16

stigtsp committed:
[nix-update-cpan] perlPackages.HTTPMessage: 6.26 -> 6.27

stigtsp committed:
[nix-update-cpan] perlPackages.IO: 1.42 -> 1.45

stigtsp committed:
[nix-update-cpan] perlPackages.IOCompress: 2.096 -> 2.100

stigtsp committed:
[nix-update-cpan] perlPackages.IOAIO: 4.73 -> 4.75

stigtsp committed:
[nix-update-cpan] perlPackages.JSONPP: 4.05 -> 4.06

stigtsp committed:
[nix-update-cpan] perlPackages.JSONParse: 0.57 -> 0.60

stigtsp committed:
[nix-update-cpan] perlPackages.libnet: 3.12 -> 3.13

stigtsp committed:
[nix-update-cpan] perlPackages.Logger: 0.037 -> 0.038

stigtsp committed:
[nix-update-cpan] perlPackages.LinkEmbedder: 1.16 -> 1.17

stigtsp committed:
[nix-update-cpan] perlPackages.JSONValidator: 4.10 -> 4.13

stigtsp committed:
[nix-update-cpan] perlPackages.JSON: 4.02 -> 4.03

stigtsp committed:
[nix-update-cpan] perlPackages.MathGMPz: 0.48 -> 0.49

stigtsp committed:
[nix-update-cpan] perlPackages.LWP: 6.49 -> 6.52

stigtsp committed:
[nix-update-cpan] perlPackages.MathBase85: 0.4 -> 0.5

stigtsp committed:
[nix-update-cpan] perlPackages.MailAuthenticationResults: 1.20200824.1 -> 2.20210112

stigtsp committed:
[nix-update-cpan] perlPackages.MacPasteboard: 0.011 -> 0.012

stigtsp committed:
[nix-update-cpan] perlPackages.LogLog4perl: 1.53 -> 1.54

stigtsp committed:
[nix-update-cpan] perlPackages.LWPProtocolHttps: 6.09 -> 6.10

stigtsp committed:
[nix-update-cpan] perlPackages.MojoliciousPluginStatus: 1.12 -> 1.13

stigtsp committed:
[nix-update-cpan] perlPackages.Mojolicious: 8.71 -> 8.73

stigtsp committed:
[nix-update-cpan] perlPackages.MojoliciousPluginOpenAPI: 3.40 -> 3.41

stigtsp committed:
[nix-update-cpan] perlPackages.Mojomysql: 1.20 -> 1.22

stigtsp committed:
[nix-update-cpan] perlPackages.MojoPg: 4.22 -> 4.24

stigtsp committed:
[nix-update-cpan] perlPackages.MinionBackendmysql: 0.21 -> 0.23

stigtsp committed:
[nix-update-cpan] perlPackages.Minion: 10.14 -> 10.15

stigtsp committed:
[nix-update-cpan] perlPackages.ModuleScanDeps: 1.29 -> 1.30

stigtsp committed:
[nix-update-cpan] perlPackages.NetHTTP: 6.19 -> 6.20

stigtsp committed:
[nix-update-cpan] perlPackages.NetSFTPForeign: 1.91 -> 1.93

stigtsp committed:
[nix-update-cpan] perlPackages.Moose: 2.2013 -> 2.2014

stigtsp committed:
[nix-update-cpan] perlPackages.perlldap: 0.66 -> 0.68

stigtsp committed:
[nix-update-cpan] perlPackages.PathTiny: 0.114 -> 0.118

stigtsp committed:
[nix-update-cpan] perlPackages.Paranoid: 2.07 -> 2.08

stigtsp committed:
[nix-update-cpan] perlPackages.NetSSLeay: 1.88 -> 1.90

stigtsp committed:
[nix-update-cpan] perlPackages.RoleTiny: 2.001004 -> 2.002004

stigtsp committed:
[nix-update-cpan] perlPackages.PPIxQuoteLike: 0.013 -> 0.015

stigtsp committed:
[nix-update-cpan] perlPackages.Specio: 0.46 -> 0.47

stigtsp committed:
[nix-update-cpan] perlPackages.SetObject: 1.40 -> 1.41

stigtsp committed:
[nix-update-cpan] perlPackages.RpcXML: 0.80 -> 0.82

stigtsp committed:
[nix-update-cpan] perlPackages.PPIxRegexp: 0.076 -> 0.078

stigtsp committed:
[nix-update-cpan] perlPackages.PodStrip: 1.02 -> 1.100

stigtsp committed:
[nix-update-cpan] perlPackages.PerlTidy: 20201207 -> 20210111

stigtsp committed:
[nix-update-cpan] perlPackages.TestFile: 1.443 -> 1.444

stigtsp committed:
[nix-update-cpan] perlPackages.Test2Suite: 0.000138 -> 0.000139

stigtsp committed:
[nix-update-cpan] perlPackages.TemplateTiny: 1.12 -> 1.13

stigtsp committed:
[nix-update-cpan] perlPackages.StringCRC32: 2 -> 2.100

stigtsp committed:
[nix-update-cpan] perlPackages.SQLAbstract: 1.87 -> 2.000001

stigtsp committed:
[nix-update-cpan] perlPackages.TestLeakTrace: 0.16 -> 0.17

stigtsp committed:
[nix-update-cpan] perlPackages.TermReadLineGnu: 1.36 -> 1.37

stigtsp committed:
[nix-update-cpan] perlPackages.TreeDAGNode: 1.31 -> 1.32

stigtsp committed:
[nix-update-cpan] perlPackages.TestManifest: 2.021 -> 2.022

stigtsp committed:
[nix-update-cpan] perlPackages.TextTrim: 1.03 -> 1.04

stigtsp committed:
[nix-update-cpan] perlPackages.TreeSimple: 1.33 -> 1.34

stigtsp committed:
[nix-update-cpan] perlPackages.TextCSV_XS: 1.44 -> 1.45

stigtsp committed:
[nix-update-cpan] perlPackages.TieCycle: 1.225 -> 1.226

stigtsp committed:
[nix-update-cpan] perlPackages.TestMockModule: 0.175.0 -> 0.176.0

stigtsp committed:
[nix-update-cpan] perlPackages.TestOutput: 1.031 -> 1.032

stigtsp committed:
[nix-update-cpan] perlPackages.Twiggy: 0.1025 -> 0.1026

stigtsp committed:
[nix-update-cpan] perlPackages.TreeSimpleVisitorFactory: 0.15 -> 0.16

stigtsp committed:
[nix-update-cpan] perlPackages.XSParseSublike: 0.10 -> 0.11

stigtsp committed:
[nix-update-cpan] perlPackages.TypeTiny: 1.012000 -> 1.012001

stigtsp committed:
[nix-update-cpan] perlPackages.URI: 5.05 -> 5.07

stigtsp committed:
[nix-update-cpan] perlPackages.Workflow: 1.48 -> 1.51

zakame reviewed (approved):
LGTM, thanks @stigtsp for the updates!

stigtsp committed:
[nix-update-cpan] perlPackages.JavaScriptMinifierXS: 0.11 -> 0.14

stigtsp committed:
[nix-update-cpan] perlPackages.TestDiagINC: init at 0.008

stigtsp commented:
@GrahamcOfBorg build perlPackages.XSParseSublike perlPackages.Workflow perlPackages.URI perlPackages.TypeTiny perlPackages.Twiggy perlPackages.TreeSimpleVisitorFactory perlPackages.TreeSimple perlPackages.TreeDAGNode perlPackages.TieCycle perlPackages.TextTrim perlPackages.TestManifest perlPackages.TextCSV_XS perlPackages.TestOutput perlPackages.TestMockModule perlPackages.TestLeakTrace perlPackages.TestFile perlPackages.Test2Suite perlPackages.TermReadLineGnu perlPackages.TemplateTiny perlPackages.StringCRC32 perlPackages.SQLAbstract perlPackages.Specio perlPackages.SetObject perlPackages.RoleTiny perlPackages.RpcXML perlPackages.PodStrip perlPackages.PPIxRegexp perlPackages.PPIxQuoteLike perlPackages.PerlTidy perlPackages.PerlMagick perlPackages.perlldap perlPackages.PathTiny perlPackages.Paranoid perlPackages.NetSSLeay perlPackages.NetSFTPForeign perlPackages.NetHTTP perlPackages.Moose perlPackages.MojoPg perlPackages.Mojomysql perlPackages.MojoliciousPluginStatus perlPackages.MojoliciousPluginOpenAPI perlPackages.Mojolicious perlPackages.ModuleScanDeps perlPackages.MinionBackendmysql perlPackages.Minion perlPackages.MathGMPz perlPackages.MathBase85 perlPackages.MailAuthenticationResults perlPackages.MacPasteboard perlPackages.LWPProtocolHttps perlPackages.LWP perlPackages.LogLog4perl perlPackages.Logger perlPackages.LinkEmbedder perlPackages.libnet perlPackages.JSONValidator perlPackages.JSONParse perlPackages.JSONPP perlPackages.JSON perlPackages.JavaScriptMinifierXS perlPackages.IOTty perlPackages.IOSocketSSL perlPackages.IOInteractive perlPackages.IOCompress perlPackages.IOAIO perlPackages.IO perlPackages.HTTPMessage perlPackages.HTTPCookies perlPackages.Gtk2Unique perlPackages.Gtk2TrayIcon perlPackages.Gtk2GladeXML perlPackages.Graph perlPackages.GnuPGInterface perlPackages.Gnome2Wnck perlPackages.Gnome2VFS perlPackages.Gnome2Canvas perlPackages.Gnome2 perlPackages.FutureAsyncAwait perlPackages.Future perlPackages.FileLoadLines perlPackages.ExtUtilsMakeMaker perlPackages.ExtUtilsInstall perlPackages.EmailMIMEContentType perlPackages.DBIxSearchBuilder perlPackages.DBDCSV perlPackages.DevelPatchPerl perlPackages.DateTimeTimeZone perlPackages.DateTimeLocale perlPackages.DateTimeFormatW3CDTF perlPackages.DateTimeFormatStrptime perlPackages.DateTimeCalendarJulian perlPackages.CSSMinifierXS perlPackages.CryptPasswdMD5 perlPackages.CryptJWT perlPackages.CryptCBC perlPackages.CPANPLUS perlPackages.CPANPerlReleases perlPackages.Connector perlPackages.ConfigTiny perlPackages.ConfigMVP perlPackages.CompressRawZlib perlPackages.CompressRawBzip2 perlPackages.ClassObservable perlPackages.Catmandu perlPackages.CatalystViewCSV perlPackages.CarpAssertMore perlPackages.Cairo perlPackages.CacheFastMmap perlPackages.BusinessISBNData perlPackages.BusinessISBN perlPackages.ArrayCompare perlPackages.Appperlbrew perlPackages.Appcpm perlPackages.AppCLI perlPackages.AlienBuild perlPackages.TestDiagINC

Mic92 commented:
can you do the macOS testing yourself or do we need to find someone?

stigtsp commented:
> can you do the macOS testing yourself or do we need to find someone?

I can test builds on Big Sur, but any help testing these would be great ofc. :)

kvtb commented:
zero changes in `buildInputs` and `propagatedBuildInputs` ?

volth-2021-02-13 commented:
> `ExtUtils::CBuilder` since it is in core, and defined twice in perl-packages.nix TODO (0.280235 fixes compilation on darwin with XCode 12, perl 5.32.1 has 0.280234)

I included to `perlPackages` the packages which has CPAN's version newer than core's, and removed when the versions match.
So probably `ExtUtils::CBuilder` is worth reincluding.
I will try to run `cpan2nix` on top of this PR + #110654 to see what it will find.

volth-2021-02-13 commented:
1. `NetServerSSPrefork`, `NetZooKeeper`, `Tirex` are candidates to move to external files as other non-CPAN modules
2. bug: `ExtUtilsCBuilder` depends on `config.allowAliases`
```
{
....
  ExtUtilsCBuilder = buildPerlPackage {
....
 
} // lib.optionalAttrs (config.allowAliases or true) {
  ExtUtilsCBuilder = null; # part of Perl 5.28
}
```

volth committed:
perl-packages.nix: make updater's parser happier

volth-2021-02-13 commented:
`HTML-TableExtract`, `podlators`, `Perl-Tidy` which are upgrading here were pinned in `cpan2nix` as "not to upgrade"
Probably their problems have gone, they need attention

volth-2021-02-13 commented:
https://github.com/stigtsp/nixpkgs/pull/3
not tested, still building

volth committed:
perlPackages.DBIxClassHelpers: remove failing test

volth committed:
[cpan2nix] perlPackages.AlgorithmDiff: 1.1903 -> 1.201

volth committed:
[cpan2nix] perlPackages.AstroFITSHeader: cleanup

volth committed:
[cpan2nix] perlPackages.BusinessISBNData: 20210112.002 -> 20210112.004

volth committed:
[cpan2nix] perlPackages.ConfigTiny: cleanup

volth committed:
[cpan2nix] perlPackages.CryptOpenSSLX509: 1.813 -> 1.902

volth committed:
[cpan2nix] perlPackages.ExtUtilsCBuilder: 0.280234 -> 0.280235

volth committed:
[cpan2nix] perlPackages.FileCountLines: cleanup

volth committed:
[cpan2nix] perlPackages.JSONParse: 0.60 -> 0.61

volth committed:
[cpan2nix] perlPackages.MathBigIntLite: cleanup

volth committed:
[cpan2nix] perlPackages.TermReadPassword: cleanup

volth committed:
[cpan2nix] perlPackages.ImagerQRCode: cleanup

volth committed:
[cpan2nix] perlPackages.LinkEmbedder: cleanup

volth committed:
[cpan2nix] perlPackages.MusicBrainzDiscID: cleanup

volth committed:
[cpan2nix] perlPackages.ParamsUtil: cleanup

volth committed:
[cpan2nix] perlPackages.PodChecker: cleanup

volth committed:
[cpan2nix] perlPackages.TestOutput: 1.032 -> 1.033

volth committed:
[cpan2nix] perlPackages.NetSCP: cleanup

volth committed:
[cpan2nix] perlPackages.NetLDAPServer: cleanup

volth committed:
[cpan2nix] perlPackages.PodAbstract: cleanup

volth committed:
[cpan2nix] perlPackages.TestScriptRun: cleanup

volth committed:
[cpan2nix] perlPackages.TestExpect: cleanup

volth committed:
[cpan2nix] perlPackages.XMLFilterSort: cleanup

volth committed:
[cpan2nix] perlPackages.Gtk2TrayIcon: cleanup

volth committed:
[cpan2nix] perlPackages.Gtk3: 0.037 -> 0.038

volth committed:
[cpan2nix] perlPackages.NetLDAPServerTest: cleanup

volth committed:
[cpan2nix] perlPackages.FileMimeInfo: cleanup

volth committed:
[cpan2nix] perlPackages.Moo: cleanup

volth committed:
[cpan2nix] perlPackages.TestUNIXSock: cleanup

volth committed:
[cpan2nix] perlPackages.DBDCSV: 0.57 -> 0.58

volth committed:
[cpan2nix] perlPackages.IOAsync: cleanup

volth committed:
[cpan2nix] perlPackages.PackageStash: cleanup

volth committed:
[cpan2nix] perlPackages.Throwable: cleanup

volth committed:
[cpan2nix] perlPackages.ModuleInstallReadmeMarkdownFromPod: cleanup

volth committed:
[cpan2nix] perlPackages.NumberFraction: cleanup

volth committed:
[cpan2nix] perlPackages.IOAsyncSSL: cleanup

volth committed:
[cpan2nix] perlPackages.ModuleInstallReadmeFromPod: cleanup

volth committed:
[cpan2nix] perlPackages.CryptPerl: cleanup

volth committed:
[cpan2nix] perlPackages.MooXlate: cleanup

volth committed:
[cpan2nix] perlPackages.JSONValidator: cleanup

volth committed:
[cpan2nix] perlPackages.NetAsyncHTTP: cleanup

volth committed:
[cpan2nix] perlPackages.MojoliciousPluginStatus: 1.13 -> 1.14

volth committed:
[cpan2nix] perlPackages.SQLAbstractLimit: cleanup

volth committed:
[cpan2nix] perlPackages.EmailSender: cleanup

volth committed:
[cpan2nix] perlPackages.WWWTwilioAPI: cleanup

volth committed:
[cpan2nix] perlPackages.RpcXML: cleanup

dependencies:
perlPackages.ModuleLoad: init at 0.36

volth committed:
[cpan2nix] perlPackages.CryptCBC: 3.00 -> 3.01

volth committed:
[cpan2nix] perlPackages.StringDiff: cleanup

volth committed:
[cpan2nix] perlPackages.EmailStuffer: cleanup

volth committed:
[cpan2nix] perlPackages.TextDiffFormattedHTML: cleanup

volth committed:
[cpan2nix] perlPackages.Autodia: cleanup

volth committed:
[cpan2nix] perlPackages.WWWMechanize: cleanup

dependencies:
perlPackages.TestTaint: init at 1.08

volth committed:
[cpan2nix] perlPackages.PDL: cleanup

volth committed:
[cpan2nix] perlPackages.Appcpm: cleanup

volth committed:
[cpan2nix] perlPackages.TestPerlTidy: cleanup

volth committed:
[cpan2nix] perlPackages.NetPing: cleanup

volth committed:
[cpan2nix] perlPackages.PerlCriticMoose: cleanup

volth committed:
[cpan2nix] perlPackages.LaTeXML: cleanup

volth committed:
[cpan2nix] perlPackages.CLIHelpers: cleanup

volth committed:
[cpan2nix] perlPackages.DBIxClass: 0.082842 -> 0.082841

volth committed:
[cpan2nix] perlPackages.BotTraining: cleanup

volth committed:
[cpan2nix] perlPackages.Starman: cleanup

volth committed:
[cpan2nix] perlPackages.CpanelJSONXS: cleanup

dependencies:
perlPackages.PodSpellCommonMistakes: init at 1.002
perlPackages.TestCheckChanges: init at 0.14
perlPackages.TestSpelling: init at 0.25

volth committed:
[cpan2nix] perlPackages.Hailo: cleanup

volth committed:
[cpan2nix] perlPackages.Workflow: 1.51 -> 1.52

volth committed:
[cpan2nix] perlPackages.CatalystRuntime: cleanup

volth committed:
[cpan2nix] perlPackages.CatalystAuthenticationStoreLDAP: cleanup

Commit:
perlPackages: i686-linux fixes

volth-2021-02-13 commented:
It would be nice to add all `Plack-*` on the next batch, there are many small packages and we miss them

volth-2021-02-13 commented:
ofbork is unhappy about `stdenv.lib`, I will fix that in `cpan2nix`.
Could you do `s/stdenv.lib/lib/g` here? it is too simple an issue to make a PR for PR

volth-2021-02-13 commented:
https://github.com/stigtsp/nixpkgs/pull/4 to make it really bulk :)

Mic92 commented:
Is this still relevant? It looks a bit old for a massive update.

stigtsp commented:
> Is this still relevant? It looks a bit old for a massive update.

It's relevant, but outdated. Working on an updated update.

jonringer commented:
I'm removing this as a blocker for 21.11.  Doesn't feel like there's enough forward momentum on it.

stigtsp commented:
> I'm removing this as a blocker for 21.11. Doesn't feel like there's enough forward momentum on it.

I agree (and regret not having time or energy to follow trough on this update).
